frontend/src/components/DetailsPanel/index.js

```diff
@@ -1,12 +1,14 @@
-import "./index.css";
-import React, { useEffect, useState } from "react";
+import dayjs from "dayjs";
+import React, { default as React, default as React, useEffect, useState } from "react";
```

Comment:
```suggestion
import React, { useEffect, useState } from "react";
```
These two `default as React` imports are causing issues on my end, seems to work if we just keep the `React` import at the start of the line